packages/react-button/src/components/Button/stories/ButtonIcon.stories.tsx

```diff
@@ -1,16 +1,18 @@
 import * as React from 'react';
-import { CalendarMonthRegular } from '@fluentui/react-icons';
+import { CalendarMonthRegular, CalendarMonthFilled, bundleIcon } from '@fluentui/react-icons';
```

Comment:
_nit:_ Order alphabetically

```suggestion
import { bundleIcon, CalendarMonthRegular, CalendarMonthFilled } from '@fluentui/react-icons';
```